browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -3857,13 +3861,28 @@ L.CanvasTileLayer = L.Layer.extend({
 			// Cursor invalidation should take most precedence among all the scrolling to follow the cursor
 			// so here we disregard all the pending scrolling
 			this._map._docLayer._painter._sectionContainer.getSectionWithName(L.CSections.Scroll.name).pendingScrollEvent = null;
+			var correctedCursor = this._visibleCursor;
+
+			if (this._docType === 'text') {
+				// For Writer documents, disallow scrolling to cursor outside of the page (horizontally)
```

Comment:
"scrolling **the** cursor"

Reply:
Thank you for the review!

No, "to" cursor - because the cursor itself is far to the right, but the view should not scroll to its position in this situation.
I do not know how [tdf#155494](https://bugs.documentfoundation.org/show_bug.cgi?id=155494) will eventually resolve, but now it's this funny situation :-)